fe/fe-core/src/main/java/com/starrocks/analysis/ShowDbStmt.java

```diff
@@ -39,6 +39,7 @@ public class ShowDbStmt extends ShowStmt {
                     .addColumn(new Column(DB_COL, ScalarType.createVarchar(20)))
                     .build();
     private final String pattern;
+    private String db;
```

Comment:
I think this should be catalog? Do you want to show databases from specific catalog?

Reply:
> I think this should be catalog? Do you want to show databases from specific catalog?

Yes.

output is like blow

MySQL [(none)]> CREATE EXTERNAL CATALOG hive_catalog_1 COMMENT "hive_catalog" PROPERTIES("type"="hive", "hive.metastore.uris"="thrift://127.0.0.1:9083");
Query OK, 0 rows affected (0.00 sec)

MySQL [(none)]> show catalogs;
+----------------+------+--------------+
| Catalog        | Type | Comment      |
+----------------+------+--------------+
| hive_catalog_1 | hive | hive_catalog |
+----------------+------+--------------+
1 row in set (0.00 sec)
MySQL [(none)]> CREATE EXTERNAL CATALOG hive_catalog_1 COMMENT "hive_catalog" PROPERTIES("type"="hive", "hive.metastore.uris"="thrift://127.0.0.1:9083");
Query OK, 0 rows affected (0.00 sec)

MySQL [(none)]> show catalogs;
+----------------+------+--------------+
| Catalog        | Type | Comment      |
+----------------+------+--------------+
| hive_catalog_1 | hive | hive_catalog |
+----------------+------+--------------+
1 row in set (0.00 sec)

MySQL [(none)]> show databases from hive_catalog_1;
ERROR 1064 (HY000): Failed to get all databases name on thrift://127.0.0.1:9083
**hms for hive_catalog_1 is down**

MySQL [(none)]> show databases from hive_catalog_1;
+----------+
| Database |
+----------+
| default  |
| tn_test  |
| un_test  |
+----------+
3 rows in set (1.10 sec)

MySQL [(none)]> show databases from hive_catalog_1;
+----------+
| Database |
+----------+
| default  |
| tn_test  |
| un_test  |
+----------+
3 rows in set (1.10 sec)

MySQL [(none)]> show databases from hive_catalog_2;
ERROR 1064 (HY000): Can not acquire databases from hive_catalog_2, please check catalog config!
**hive_catalog_2 is not configured**